drivers/slipdev/slipdev.c

```diff
@@ -67,11 +68,90 @@ static inline void slipdev_unlock(void)
     }
 }
 
-static void _slip_rx_cb(void *arg, uint8_t byte)
+static inline void _slipdev_stdio_add_to_frame(slipdev_t * dev, uint8_t byte) {
```

Comment:
The name is a bit confusing, we don't have a stdio frame, how about just

```suggestion
static inline void _slipdev_stdin_add(slipdev_t * dev, uint8_t byte)
{
```